llvm/include/llvm/CodeGen/MachineFunction.h

```diff
@@ -293,6 +293,10 @@ class LLVM_EXTERNAL_VISIBILITY MachineFunction {
   // numbered and this vector keeps track of the mapping from ID's to MBB's.
   std::vector<MachineBasicBlock*> MBBNumbering;
 
+  // MBBNumbering epoch, incremented after renumbering to detect use of old
+  // block numbers.
+  unsigned MBBNumberingEpoch;
```

Comment:
```suggestion
  unsigned MBBNumberingEpoch = 0;
```
Maybe I missed it somewhere, but I think this is uninitialized?

Reply:
Good catch!